app/lib/benchmark/benchmark_grid.html

```diff
@@ -26,4 +31,4 @@
         [data]="benchmark"
         (onZoomIn)="zoomInto = benchmark">
     </benchmark-card>
-</div>
\ No newline at end of file
+</div>
```

Comment:
nit: the project leaves a new line at EOF

Reply:
Added.